### PR TITLE
Link 'My snaps' to new account snaps page

### LINKED ISSUE #1073: Redirect 'My snaps' to snapcraft.io/account/snaps

### PR DESCRIPTION
**Pull request by bartaz**
## Done

Update the 'My snaps' link in header to new account snap page (same as on snapcraft.io)

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- 'My snaps' link in header should not have external icon
- 'My snaps' link in header should link to snapcraft.io/account/snaps

## Issue / Card

Fixes #1073 

## Screenshots

screen shot 2018-04-30 at 07 19 06
screen shot 2018-04-30 at 07 19 23

### CODE REVIEW

**Comment by Greggless** [comment]
We've gone backwards and forwards with where 'My snaps' is going to point to. I [originally specc'd it ](https://docs.google.com/document/d/1NSKrHD5JTFw-nA2bZHRnOr9nlu6sXyCRNJAQzqrhIYI/edit#heading=h.n2on9itjkmhh) to point to Dashboard until snapcraft.io was more useful. That was changed as we decided to point to **snapcraft.io**. Who is making the final decision on this? @therealjuan @carlaberkers @matthewpaulthomas ?

**Comment by carlaberkers** [comment]
heard you had a good conversation about this, @therealjuan can you post the outcome, let @bartaz know, and make sure the right pull requests are closed / committed? Thanks!

**Comment by matthewpaulthomas** [comment]
From [snapcraft.io#595](https://github.com/canonical-websites/snapcraft.io/issues/595):
> Summary from meeting this morning:
> - snapcraft.io/account being much less useful than Dashboard is less of an issue for people who haven’t used Dashboard before.
> - A bigger problem is that, with the “Store” pages becoming more popular, end users have been creating Dashboard accounts in the expectation that “My snaps” will show what snaps they have installed. ([I predicted last August](https://docs.google.com/document/d/1FkVSByHbSQljDXLw215GSWt21GFUip9SxO-NeG9Vrmk/edit#heading=h.anri58a7vnzs) that “Combining developer and end-user sites would make the top-level navigation confusing for end users”.)
> - To mitigate this, we should investigate replacing the “My snaps” link with a landing page that describes the publishing process and the various UIs. I’m tracking that work as [snapcraft-design#438](https://github.com/CanonicalLtd/snapcraft-design/issues/438).

**Comment by therealjuan** [comment]
Just to provide more context:

* Only developers that created their accounts with dashboard will find the new account pages being less useful. For now developers as stated by @matthewpaulthomas they will not be aware of the old features. Having a look at the data on the last 4 months in Google Analytics, only ~2% of users ever visited (not used) the Register a Snap page

* Talking with the advocacy team, we discussed the possibility to communicate old users of snapcraft.io about old features being migrated from dashboard to the new account pages during the coming months, so we can use this as an opportunity to raise awareness and feedback from the field

* Having **My Snaps** currently as a top level navigation item is confusing for end users as its intended to publishers. We should explore the option of including *My published Snaps* (or the term we will use) inside the logged-in account dropdown navigation

In a nutshell, we can close this pull request and decided an alternative for **My Snaps** and its placement in the navigation

**Comment by matthewpaulthomas** [comment]
Ok, closing this PR per @therealjuan. I’ve posted a proposed mini-roadmap in [snapcraft.io#595](https://github.com/canonical-websites/snapcraft.io/issues/595).
